mne/decoding/tests/test_time_gen.py

```diff
@@ -70,7 +70,15 @@ def test_generalization_across_time():
     assert_equal("<GAT | fitted, start : -0.200 (s), stop : 0.499 (s), no "
                  "prediction, no score>", '%s' % gat)
     assert_equal(gat.ch_names, epochs.ch_names)
+    # test different predict function:
+    gat.decision_function(epochs)
+    assert_array_equal(np.shape(gat.y_pred_), (15, 15, 14, 1))
+    gat.predict_proba(epochs)
+    assert_array_equal(np.shape(gat.y_pred_), (15, 15, 14, 2))
```

Comment:
ok I slept over it...

do as you feel. You know this code better than me know.

to have a sklearn like API we would need a GatResults object with a plot method that is obtained from predict or predict_proba. But it would not be a numpy array so not really sklearn anyway.

if @jona-sassenhagen is happy I don't want to block things here